client/index.js

```diff
@@ -68,7 +83,7 @@ var newConnection = function() {
 	}));
 
 	sock.onclose = function() {
-		console.error("[WDS] Disconnected!");
+		log("error", "[WDS] Disconnected!");
```

Comment:
However this is generally not useful at all, especially because it enters a crazy loop where it prints this message every few seconds:

![](https://cloud.githubusercontent.com/assets/810438/17225095/757b8a10-54fb-11e6-93ef-a335281292f3.png)